src/entry-client.js

```diff
@@ -42,19 +42,26 @@ function perfInit(to, from) {
 // 'start' mark
 export function perfMeasure(name) {
     if (!perfEnabled()) {
-        return null;
+        return false;
     }
 
     const mark = getCurrentPerfMark();
     if (!mark) {
         // Can't measure if we don't have a starting mark to measure from
-        return null;
+        return false;
     }
 
     // Add a measurement from the start mark with the current name.  Example:
     //     urbnperf|Homepage->Catch-All|done
     const [prefix, route] = mark.name.split('|');
-    return window.performance.measure(`${prefix}|${route}|${name}`, mark.name);
+    window.performance.measure(`${prefix}|${route}|${name}`, mark.name);
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure this is not supposed to return anything